### ISSUE TITLE
Drop Item Formulae Elderfolk

### ISSUE THREAD

**Issue by Melkkart** [AnthonyMonette/TheWitcherTRPG#319]
When dragging it to the character sheet, it does not appear in the inventory, but it really is. 

Formula type ... Elderfolk Armor: Not visible

Formula type ... Ammunition (Journeyman): Not visible.

But I'm not sure why, for example some objects marked as Toolkit appear and others do not. It is a strange bug.
My game has a long time, it is possible that so many changes over time, have affected it. 

![image](https://github.com/AnthonyMonette/TheWitcherTRPG/assets/123239719/13b83fb7-7114-42f1-8fe4-8767dcd1ff57)

None of these are visible, but they are there. 

The Actor
[fvtt-Actor-prueba-d3mGJkljnBrgW86k.zip](https://github.com/AnthonyMonette/TheWitcherTRPG/files/12473383/fvtt-Actor-prueba-d3mGJkljnBrgW86k.zip)

**Comment by AnthonyMonette** [comment]
I think I also saw this issue on my side while doing some tests.  
I'm gonna do some more investigation but I don't think it's only in your game

**Comment by AnthonyMonette** [comment]
Okay I just found the issues for the diagrams, but in your case it seems to be specific objects that are problematics, am I right?

**Comment by AnthonyMonette** [comment]
and Which version are you using?

**Comment by Melkkart** [comment]
Yes, it has happened to me with Toolkits, and ingredient, do you want me to do some specific test?
I use Witcher 0.97 and Foundry v11 Build 307

**Comment by Melkkart** [comment]
Ok, I have removed the modules, and it seems that everything is fine, except for the formulas, I have also created a new character.
It seems to be just the Elderfolk.

I just found the bug, or so I think. 
Could it be because of the length of the text? 
I have dragged an object that its name was: "Habitación de Posada Media (Por Noche)"
It has not appeared on the character sheet.  So I have renamed it to "Habitación de Posada Media"
and when dragged it is visible.
I think they are going to be excessively long names.

**Comment by AnthonyMonette** [comment]
For the formulae and diagrams it will be fixed in the next release.  For the other use-case:
If you press F12, then Drag n Drop the one with the biggest name, can you tell me if any error occurs in the console?

**Comment by Melkkart** [comment]
It's Strange, now it has dragged me long names, and others not.
![image](https://github.com/AnthonyMonette/TheWitcherTRPG/assets/123239719/1052e0ff-2ac6-455c-996d-d9b6c51adb67)
I do not see any error
![image](https://github.com/AnthonyMonette/TheWitcherTRPG/assets/123239719/8d1996ec-ae06-4182-8452-bca3cfca52d7)

I'm with the basics. no modules. And English version.
[fvtt-Actor-prueba-d3mGJkljnBrgW86k.zip](https://github.com/AnthonyMonette/TheWitcherTRPG/files/12477128/fvtt-Actor-prueba-d3mGJkljnBrgW86k.zip)
